README.md

```diff
@@ -1,15 +1,41 @@
-zj-58
-=====
+ZJ-58 (and ZJ-80)
+=================
 
-CUPS filter for thermal printer Zjiang ZJ-58
+CUPS filter for thermal printer Zjiang ZJ-58 and JZ-80 with cutter.
+These drivers are also work with chinese XPrinters models XP-58, XP-80, XP-N160II, etc. And I think it will work with almost any ESC/POS printers.
+
+Preamble
+--------
+
+This repository is based on https://github.com/klirichek/zj-58 which is (at least for now) not maintained.
```

Comment:
When you open a PR to said repo then this line may not be reasonable to put here. However when you fork the repo it will be fine i guess :)

Reply:
Wow, sorry, i have forked repo and made changes just for myself. Useful changes for pull request are only in C sources...